projects/plugins/jetpack/extensions/blocks/subscriptions/deprecated/v6/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import definedAttributes from '../v3/attributes';
+import save from './save';
+
+/**
+ * Deprecation reason
+ *
+ * Added a new input field successMessage.
+ */
+export default {
+	attributes: definedAttributes,
+	migrate: attributes => {
```

Comment:
I'm not thinking there needs to be a `migrate` method here. We aren't needing to modify previous attributes, we are simply adding a new attribute. Older block versions should still validate/save, and then they can optionally use the new `successMessage`. @jeherve Does that match your understanding when a new attribute is being added?

Reply:
I got your idea, what are you're trying to say... Overall as far as i understand deprecation `v6` is nothing but current version of block master branch. Once my code gets merged it will be considered as new/updated version of the block. 

`migrate` method is only required when we are updating/changing existing block attribute. If we are adding a new block attribute we don't need `migrate` method.

Reply:
> migrate method is only required when we are updating/changing existing block attribute. If we are adding a new block attribute we don't need migrate method.

Right, unless for some reason we needed to alter past deprecated versions to include that new attribute. In this case `successMessage` has a default value and can optionally be customized. So the past versions don't need to include the new attribute in their migrate methods.